editor/modes/visual-editor/block-list.js

```diff
@@ -106,9 +101,8 @@ export default connect(
 		selectionEnd: getBlockSelectionEnd( state ),
 	} ),
 	( dispatch ) => ( {
-		clearSelectedBlock: () => dispatch( { type: 'CLEAR_SELECTED_BLOCK' } ),
 		onMultiSelect( { start, end } ) {
 			dispatch( { type: 'MULTI_SELECT', start, end } );
 		},
 	} )
-)( clickOutside( VisualEditorBlockList ) );
```

Comment:
Does this need a decision somehow? I'm a bit confused on the details, but I'd like to help work through the flow if need be, and if someone can clarify the conflict.